sdx_controller/controllers/l2vpn_controller.py

```diff
@@ -223,14 +219,14 @@ def patch_connection(service_id, body=None):  # noqa: E501
 
     logger.info(f"Gathered connexion JSON: {new_body}")
 
-    body = json.loads(value[service_id])
+    body = value[service_id]
```

Comment:
@congwang09, what do you think about encapsulating the dict key access on the read_from_db call? Although it is a simple change, we end up having to duplicate the code above everywhere that we read from the DB.

Reply:
Sounds good. I added a new function to only get the value.